### PR TITLE
Upgrade dependencies

### LINKED ISSUE #4: Dependencies are outdated
Dependencies are outdated, and as such, they should probably be updated. The listed version of Tape also includes a security vulnerability. As for the gonzales-pe update, #1 should have introduced a fix to ease upgrading.

I'll start with gonzales-pe first.

### PR DESCRIPTION
**Pull request by jorgeavaldez**
Closes #4

### CODE REVIEW

**Comment by jorgeavaldez** [comment]
Tests ran after gonzales update

File: bin/sass-flatten [hunk 1]
```diff
@@ -9,26 +9,42 @@ var write = fs.writeFileSync;
...
+  return;
+}
+
 var inc = cli.flags.includePath || path.dirname(src);
 
 if (src) {
   write(dest, flatten(read(src, 'utf8'), inc));
 } else {
-  stdin(function (data) {
+  getStdin().then(function (data) {
```
Comment (jorgeavaldez) [inline bin/sass-flatten:39]: `meow` now returns a promise

File: bin/sass-flatten [hunk 2]
```diff
@@ -9,26 +9,42 @@ var write = fs.writeFileSync;
...
+Options
+  --include-path            Path to look for imported files
+`, {
+  pkg: require('../package.json')
 });
 
 var src = cli.input[0];
 var dest = cli.input[1] || src;
+
+if (!src) {
```
Comment (jorgeavaldez) [inline bin/sass-flatten:29]: Before, this would break on `path.dirname(src)` since `src` would be undefined if not passed in.

**Comment by cykacookie** [comment]
lgtm

**Review by cykacookie — APPROVED** [review]
👍

**Pull request merged**